### PR TITLE
Fix wmarchive format

### PR DESCRIPTION
**Pull request by ticoann**
I will add ArchiveStatus poller code which will control the rate of the upload. 1000 each polling cycle and 100 bulk update. so It can be patched 1.0.14 if necessary later. 

Also need to create patch for wmagent 1.0.14 branch

### CODE REVIEW

**Comment by ticoann** [comment]
@yuyiguo, Yuyi I updates the conversion function which handle various different type of fwjr including LogCollect job report. This might cover all the cases, but we will update as we find the broken cases.

You can use this function to update Mongo db. Just need to filter out fwjr['jobtype'] == "Cleanup".
We can discuss more detail on Monday

File: test/python/WMCore_t/Services_t/WMArchiver_t/FWJRSamples/LogCollectSuccessFwjr.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                   "analysis": [
+                   ],
+                   "LogCollect": [
+                       {
+                           "runs": {
+                           },
+                           "SEName": "srm-cms.cern.ch",
+                           "lfn": "srm://srm-cms.cern.ch:8443/srm/managerv2?SFN=/castor/cern.ch/cms/store/logs/prod/2016/03/WMAgent/sryu_DQMHarvesting_Sryu_reqmgr2_160318_050142_4405/sryu_DQMHarvesting_Sryu_reqmgr2_160318_050142_4405-EndOfRunDQMHarvestLogCollect-b6d9613445-1-logs.tar",
+                           "location": null,
+                           "PNN": "T2_CERN_CH",
```
Comment (amaltaro) [inline test/python/WMCore_t/Services_t/WMArchiver_t/FWJRSamples/LogCollectSuccessFwjr.json]: T2_CH_CERN

**Comment by ticoann** [comment]
still need to do #6573 and #6802

**Comment by ticoann** [comment]
test this please

**Pull request merged**